src/PeopleTable.tsx

```diff
@@ -0,0 +1,127 @@
+import cn from 'classnames';
+import { useEffect, useState } from 'react';
+import { Loader } from './components/Loader';
+import { Person } from './types/Person';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from './PersonLink';
+
+function getExtendedPersonList(data: Person[]): Person[] {
+  return data.map(person => {
+    return {
+      ...person,
+      mother: data.find(curPerson => person.motherName === curPerson.name),
+      father: data.find(curPerson => person.fatherName === curPerson.name),
+    };
+  });
+}
+
+export const PeopleTable = () => {
+  const [data, setData] = useState<Person[]>([]);
+  const [isloading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  const { personSlug } = useParams();
+  const selectedPersonSlug = personSlug ? personSlug : null;
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(
+          'https://mate-academy.github.io/react_people-table/api/people.json',
+        );
+        const jsonData = await response.json();
+
```

Comment:
You have  function getPeople() in api.ts. So you just can use it here

Reply:
Good point, fix this